### PR TITLE
Update admin RPC endpoints

### PR DESCRIPTION
**Pull request by antonydenyer**
### What does this PR do?

Expose more NodeInfo properties from admin_nodeInfo RPC endpoint
Wire up admin_peers RPC endpoint

### CODE REVIEW

File: core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java [hunk 1]
```diff
@@ -10,48 +10,81 @@
...
-    public static class Result {
-        public String getName() {
-            return name;
-        }
-
+    public static class NodeInfo {
+        private String enode;
+        private String id;
+        private String ip;
+        private String lisstenAddr;
```
Comment (iikirilov) [inline core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java]:
```suggestion
        private String listenAddr;
```

File: core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java [hunk 2]
```diff
@@ -10,48 +10,81 @@
...
         private String name;
 
-        Result(String name) {
+        public NodeInfo() {}
+
+        public NodeInfo(String enode, String id, String ip, String lisstenAddr, String name) {
+            this.enode = enode;
+            this.id = id;
+            this.ip = ip;
+            this.lisstenAddr = lisstenAddr;
```
Comment (iikirilov) [inline core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java]:
```suggestion
            this.listenAddr = listenAddr;
```

File: core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java [hunk 3]
```diff
@@ -10,48 +10,81 @@
...
+
+        public String getId() {
+            return id;
+        }
+
+        public String getIp() {
+            return ip;
+        }
+
+        public String getLisstenAddr() {
```
Comment (iikirilov) [inline core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java]:
```suggestion
        public String getListenAddr() {
```

File: core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java [hunk 4]
```diff
@@ -10,48 +10,81 @@
...
+            return lisstenAddr;
```
Comment (iikirilov) [inline core/src/main/java/org/web3j/protocol/core/methods/response/admin/AdminNodeInfo.java]:
```suggestion
            return listenAddr;
```

**Comment by antonydenyer** [comment]
> also some fields are missing from the node info and peers

I'm a bit wary of adding too many things in, there are no standards around this yet so need to do some more analysis before adding more fields in.

**Pull request merged**